plugins/pa11y/readme.md

```diff
@@ -0,0 +1,30 @@
+# @dotcom-tool-kit/pa11y
+
+A plugin to run [Pa11y](https://github.com/pa11y/pa11y) accessibility tests. This plugin uses [Pa11y CI](https://github.com/pa11y/pa11y-ci) to run the tests.
+
+## Installation
+
+With Tool Kit [already set up](https://github.com/financial-times/dotcom-tool-kit#installing-and-using-tool-kit), install this plugin as a dev dependency:
+
+```sh
+npm install --save-dev @dotcom-tool-kit/pa11y
+```
+
+And add it to your repo's `.toolkitrc.yml`:
+
+```yaml
+plugins:
+    - '@dotcom-tool-kit/pa11y'
+```
+
+## Options
+
+| Key | Description | Default value |
+|-|-|-|
+| `configFile` | Path to the config file | `.pa11yci.js` |
+
+## Hooks
```

Comment:
**suggestion**: perhaps we could change this title to `Default Hooks` instead, to make it clear that users can override this hook with another task

Reply:
I prefer something like `Preconfigured Hooks` as it's something you override in your own config rather than something you can modify in the plugin's config...? idk

Reply:
yes let's go for `Preconfigured Hooks` - it's a more accurate term in this context